### DISCUSSION TITLE
Not sure if using the correct PIA encryption level with the new settings merge

### DISCUSSION THREAD

**Discussion by MrJamesT** [qdm12/gluetun#786] [Q&A] [answered]
I am using PIA and ever since I started using this package I had to switch to "normal" encryption level because my poor synology nas was utilized to 100% when downloading something over the vpn on strong encryption. After I updated the package recently, I haven't changed the settings and the container started reporting as "unhealthy" because it couldn't reach the healthcheck server.
I figured out this is due to me only forwarding UDP port 1198 through firewall previously, now it's using TCP 501, could this be because the new settings merge caused an issue and it switched back to "strong" encryption?

**Comment by qdm12** [comment]
Yes it might be. I had a quick check and it looks fine. What encryption preset do you have logged in the settings tree?

**Reply by MrJamesT** [reply]
Oh, while looking at my docker-compose I've noticed that I have `OPENVPN_PROTOCOL=tcp` along with `PIA_ENCRYPTION=normal`, even though it was there previously, the connection was still through UDP. Could it be that the update fixed the option for the TCP protocol  that didn't work before?

**Reply by qdm12** [reply]
Oops no idea 😄 !! What was wrong before? Was it not taking the `OPENVPN_PROTOCOL=tcp` into account when using the PIA encryption preset to normal??

**Comment by MrJamesT** [comment, marked as answer]
Yes I think so, anyway, I checked my netdata logs and the usage seems to not be 100% when downloading so it looks like the encryption preset is correct :). Thank you for checking into it, TCP or UDP, I'm happy it works! ;)

**Reply by qdm12** [reply]
Awesome! Happy tunneling!
